api/src/main/java/javax/enterprise/concurrent/package-info.java

```diff
@@ -15,6 +15,6 @@
  */
 
 /**
- * Classes and interfaces for concurrent programming in Java EE&trade; applications. 
+ * Classes and interfaces for Jakarta EE Concurrency applications. 
```

Comment:
Was this intended to be `concurrency in Jakarta EE applications`?

Reply:
it wasn't but it might read better that way